CONTRIBUTING.md

```diff
@@ -54,5 +54,6 @@ See our [security disclosure instructions](.github/SECURITY.md) for more details
 
 ## Licensing
 
-See the [LICENSE](LICENSE) file for our project's licensing. We will ask you to confirm the licensing of your contribution.
-
+Kani is distributed under the terms of both the MIT license and the Apache License (Version 2.0).
+See [LICENSE-APACHE](LICENSE-APACHE) and [LICENSE-MIT](LICENSE-MIT) for details.
+We will ask you to confirm the licensing of your contribution.
```

Comment:
Can we improve this line? This makes me think that there will be a check box somewhere.

Reply:
Yeah, I know that's there and that's why I was thinking that the text should be somewhat similar here.

Reply:
Done